loxt.js.org

AngelNext opened the pull request:
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your domain request, you need to create a file named `CNAME` in the root of your github pages site (_this may not be the root of your repository_). 📝
This file should contain a single line with the full js.org domain you are requesting, in this instance `loxt.js.org`.

AngelNext commented:
Should be done now!

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But CNAMES are still missing. You have to (re-)add a file named ```CNAME``` to your branch with the single line ```loxt.js.org``` to make it work (or use the repo settings to do so)
I'll merge after we have "contact". Please keep this PR open.

AngelNext commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But CNAMES are still missing. You have to (re-)add a file named `CNAME` to your branch with the single line `loxt.js.org` to make it work (or use the repo settings to do so) I'll merge after we have "contact". Please keep this PR open.

I added the CNAME to the gh-pages branch and to the main branch, in fact, is you go to https://angelnext.github.io/loxt-docs/CNAME you will see it.

indus commented:
That my cause the problem. GH only supports one CNAME. The first one evaluated blocks the second. I suggest you first remove both and only add the one where your page is located. That triggers a re-evaluationAm 01.08.2022 17:17 schrieb AngelNext ***@***.***>:

I've already added your requested subdomain to JS.ORGs zonefile. But CNAMES are still missing. You have to (re-)add a file named CNAME to your branch with the single line loxt.js.org to make it work (or use the repo settings to do so) I'll merge after we have "contact". Please keep this PR open.

I added the CNAME to the gh-pages branch and to the main branch, in fact, is you go to https://angelnext.github.io/loxt-docs/CNAME you will see it.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

indus commented:
It is still not working (and I can see a second CNAME in the public folder ?!?). You havn't removed the one in the root of the master branch and re-added it, right? Without doing so there is no re-evaluation by GitHub and it wont work.

indus commented:
@AngelNext  any updates?

AngelNext commented:
Yes, I deleted all CNAME files in my repository and re-added the CNAME file in the root, but should it be in the main root, or in the pages root, note that I don't have a gh-pages, I use GitHub actions for publishing to pages. And would adding the domain in the repo settings make the connection? I don't know a lot about this.

MattIPv4 commented:
I would assume the build output just needs to include the CNAME file so that GitHub Pages detects it.

indus commented:
Here is a comment for teh action regarding CNAME files: https://github.com/marketplace/actions/deploy-to-github-pages#additional-build-files-
@AngelNext: I guess you have to figure this out by your own. https://angelnext.github.io/loxt-docs/ will redirect to the JS.ORG domain when settings are correct.

AngelNext commented:
I have added the domain in pages settings, It tells me that the DNS check was successful, I think the connection should be already done!

indus commented:
@AngelNext: Now we have a redirect error...
I think your setup is too erratic for this service and for me the problems are no longer traceable. I would recommend to end this here.

AngelNext commented:
Ye I've also had the error, But I figured it was on your side, because it also happened with https://loxt.js.org

MattIPv4 commented:
I'm not sure how a redirect error would be on our side, we're just providing a DNS record, no redirects etc.

You likely have something in your code that is redirecting the user to your old github.io URL, which is then redirecting back to the js.org URL, and looping...

indus commented:
@AngelNext As there is no real progress I'm going to close this.
